builds/jazz-build-module/sls-app/serverless-build-rules.yml

```diff
@@ -354,8 +354,11 @@ provider:
   iamManagedPolicies: # Optional IAM Managed Policies, which allows to include the policies into IAM Role
     sbr-type: "[arn-iam-policy]"
     sbr-render: merge
+    sbr-constraint:
+        sbr-whitelist: iamManagedPolicies
```

Comment:
Have you validated the following cases:
1. No user managedPolicies and some config managedPolicies
2. No config managedPolicies and some user managedPolicies
3. No user managedPolicies and no config managdPolicies
4. Some user managedPolicies and some config managedPolicies (with some common and some not common)

Reply:
I have validated all the four cases and it works fine as expected. 

-  For 1 and 2, since no user managedpolicies, it wont come in effect because  sbr-mandatory is not set to true
- For 4 , If any outstanding it will throw error otherwise build will works
- For 3,  It will throw error as expected, since it is merge it should expect both config and userval must be there